core/core-env.el

```diff
@@ -63,7 +63,7 @@ current contents of the file will be overwritten."
           "# regexps in `spacemacs-ignored-environment-variables'. If you add any\n"
           "# duplicate settings for a variable, only the first setting is effective.\n"
           "# PATH is a special case: all PATH settings are read, each non-duplicate\n"
-          "# directory entry is appended to the `exec-path' variable, and then PATH is\n"
+          "# directory entry is prepended to the `exec-path' variable, and then PATH is\n"
```

Comment:
@smile13241324 I have also updated the generated documentation here to reflect this change